config/services.yaml

```
@@ -6,6 +6,78 @@
+    tags:
+      - { name: 'api_platform.filter', id: 'customer.mongodb.type.partial_search' }
+
+  app.customer_status.order_filter:
+    parent: 'api_platform.doctrine_mongodb.odm.order_filter'
+    arguments: [{ 'ulid': 'desc', 'value': 'asc' }]
+    tags:
+      - { name: 'api_platform.filter', id: 'customer.mongodb.status.order' }
+
+  app.customer_status.partial_search_filter:
+    parent: 'api_platform.doctrine_mongodb.odm.search_filter'
+    arguments: [{ 'value': 'partial' }]
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)